imap_processing/mag/l1a/mag_l1a_data.py

```diff
@@ -46,84 +48,95 @@ def __add__(self, seconds: float):
 
         return TimeTuple(coarse, fine)
 
+    def to_seconds(self):
+        """
+        Convert time tuple into seconds (float).
 
-@dataclass
-class Vector:
-    """
-    Data class for storing MAG vector data.
-
-    Attributes
-    ----------
-    timestamp : TimeTuple
-        Time of the vector sample
-    vectors : tuple[int, int, int, int]
-        Vector sample, containing x,y,z,range
-    """
-
-    # TODO: This timestamp should be in J2000/datetime64
-    timestamp: TimeTuple
-    x: int
-    y: int
-    z: int
-    rng: int
-
-    def __init__(self, vectors, time: TimeTuple):
-        self.timestamp = time
-        self.x = vectors[0]
-        self.y = vectors[1]
-        self.z = vectors[2]
-        self.rng = vectors[3]
+        Returns
+        -------
+        seconds: float
+            Time in seconds
+        """
+        return self.coarse_time + self.fine_time / MAX_FINE_TIME
 
 
 @dataclass
 class MagL1a:
     """
     Data class for MAG Level 1A data.
 
+    TODO: This object should end up having 1 days worth of data.
```

Comment:
This is mostly true right now, but I'm leaving some of the work for when SPICE kernels get involved.